test/uk/gov/hmrc/tai/service/IncomeServiceSpec.scala

```diff
@@ -98,12 +98,46 @@ class IncomeServiceSpec extends BaseSpec {
           BigDecimal(0))
       )
 
+      val employment = Employment(
+        "company name",
+        Live,
+        Some("888"),
+        new LocalDate(TaxYear().next.year, 5, 26),
+        None,
+        Nil,
+        "",
+        "",
+        1,
+        Some(100),
+        hasPayrolledBenefit = false,
+        receivingOccupationalPension = true
+      )
+      val employment2 = Employment(
+        "company name",
+        Live,
+        Some("888"),
+        new LocalDate(TaxYear().next.year, 5, 26),
+        None,
+        Nil,
+        "",
+        "",
+        2,
+        Some(100),
+        hasPayrolledBenefit = false,
+        receivingOccupationalPension = true
+      )
+
+      val mockEmploymentService = mock[EmploymentService]
       val mockIncomeRepository = mock[IncomeRepository]
       when(mockIncomeRepository.taxCodeIncomes(any(), any())(any()))
         .thenReturn(Future.successful(taxCodeIncomes))
 
-      val SUT = createSUT(incomeRepository = mockIncomeRepository)
-      val result = SUT.taxCodeIncomes(nino, TaxYear())(HeaderCarrier()).futureValue
+      when(mockEmploymentService.employments(any[Nino], any[TaxYear])(any[HeaderCarrier]))
+        .thenReturn(Future.successful(Seq(employment, employment2)))
+
+      val sut = createSUT(employmentService = mockEmploymentService, incomeRepository = mockIncomeRepository)
```

Comment:
We can remove employmentService = mockEmploymentService as createSut has set this by default

Reply:
I wouldn't do it just because we're setting the behaviour for the `employmentService`. The other way would be leaving the default and then accessing the mock to modify it. Because of this I would just follow what's been done in other tests which is what u can see above